x-pack/plugins/cloud_security_posture/public/test/mock_server/constants.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const MOCK_SERVER_BASE_URL = 'http://localhost';
+export const MOCK_SERVER_LICENSING_INFO_URL = `${MOCK_SERVER_BASE_URL}/api/licensing/info`;
```

Comment:
what about having it closer to the `licensing` handler? Right now the URLs are all over the place, I don't see much value in having "domain" specific constants on the mock_server level

Reply:
Well put, that makes a lot of sense!